jest.config.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
     "packages/reakit/src/**/*.{js,ts,tsx}",
     "!**/*-test.{js,ts,tsx}"
   ],
-  projects: ["<rootDir>/packages/*/jest.config.js"],
+  projects: ["<rootDir>/packages/*/jest*config.js"],
```

Comment:
Was this a deliberate change?

Reply:
@jperasmus Yes! I wanted it to consider files like `jest.safari.config.js`.

Reply:
Alrighty, thanks.